### PR TITLE
Fixed bug CTS-1806 TC dev for RTCPeerConnection for Chromium 51

### PR DESCRIPTION
**Pull request by kaixinjxq**
BUG=https://crosswalk-project.org/jira/browse/CTS-1806
Impacted tests(approved): new 1, update 0, delete 5
Unit test platform: Crosswalk for Android 21.51.544.0
Unit test result summary: pass 1, fail 0, block 0

### CODE REVIEW

File: webapi/webapi-webrtc-w3c-tests/tests.full.xml [hunk 1]
```diff
@@ -485,6 +425,18 @@
           </spec>
         </specs>
       </testcase>
+      <testcase component="W3C_HTML5 APIs/Communication/WebRTC" execution_type="auto" id="RTCPeerConnection-EventTarget" priority="P1" purpose="Test RTCPeerConnection Data-Only Connection Test with Promises" status="approved" type="compliance" subcase="1">
```
Comment (Honry) [inline webapi/webapi-webrtc-w3c-tests/tests.full.xml]: 'subcase=1' is unnecessary

**Comment by Honry** [comment]
Could you explain why remove these 5 tests?

**Comment by kaixinjxq** [comment]
Add promise-based versions of RTCPeerConnection methods: setLocalDescription, setRemoteDescription, addIceCandidate, createOffer and createAnswer on Chromium 51.
The new cover the five method, so delete the five old callback files.

**Comment by Honry** [comment]
Thanks, LGTM.

**Pull request merged**